extensions/lifecycle/conductor/service.js

```diff
@@ -59,7 +59,7 @@ function handleMetrics(res, log) {
 }
 
 async.waterfall([
-    done => startProbeServer(config.healthcheckServer, (err, probeServer) => {
+    done => startProbeServer(lcConfig.conductor.probeServer, (err, probeServer) => {
```

Comment:
will this config require joi changes?

Reply:
they all fall under extensions and schema for extensions is just, `joi.object()`

Reply:
oh... well that is both good and bad lol